sklearn/model_selection/_split.py

```diff
@@ -862,6 +864,10 @@ def _iter_test_masks(self, X, y, groups):
             raise ValueError("The groups parameter should not be None")
         groups = np.array(groups, copy=True)
         unique_groups = np.unique(groups)
+        if self.n_groups > len(unique_groups):
+            raise ValueError("Cannot have n_groups=%d greater than the number "
+                             "of unique groups: %d."
```

Comment:
I copied the format from this other [error](https://github.com/polmauri/scikit-learn/blob/5a37b37958438207393d93fa6774cc441a97ccb2/sklearn/model_selection/_split.py#L868) in the same file, should I change it?